storagenode/storagenodedb/storagenodedbtest/multidbsnapshot.go

```diff
@@ -0,0 +1,86 @@
+package storagenodedbtest
+
+import (
+	"fmt"
+	"sort"
+
+	"storj.io/storj/internal/dbutil/dbschema"
+	"storj.io/storj/internal/dbutil/sqliteutil"
+)
+
+// Snapshots is the global variable that stores all the snapshots for testing
+var Snapshots MultiDBSnapshots
+
+// MultiDBSnapshots provides a convenient list of MultiDBSnapshot
+type MultiDBSnapshots struct {
+	List []*MultiDBSnapshot
```

Comment:
Why not a `map[int]*MultiDBSnapshot`? Then we wouldn't need to sort and lookup would be constant time.